### DISCUSSION TITLE
Boşluk bırakma hatası

### DISCUSSION THREAD

**Discussion by ElifAltunsu** [19/classroom#42] [Anonim] [answered]
![error 1](https://user-images.githubusercontent.com/72800795/159130103-b950104a-7c41-448f-9fb7-3af505f77cca.jpg)
![error 2](https://user-images.githubusercontent.com/72800795/159130105-1170d4dd-0f11-4000-98cd-9597383f2fba.jpg)

Ödevimi GitHub üzerinden yapıyorum ve otomatik notlandırma sisteminde böyle bir sorun alıyorum. Tanımlamalar arası bir satır boşluk bırakarak gönderince "trailing whitespaces detected" sorununu, satır boşluğu bırakmadan gönderince de "Expected 1 empty line between definitions" sorunu alıyorum.  Ödevin son halini derste göstermiş olduğunuz üzere boşluklu olarak bıraktım.

**Comment by ElifAltunsu** [comment]
Hocam, kusura bakmayın. Satır boşluklarındaki space boşluklarını silmeyi unutmuşum. Sorun halledildi

**Comment by roktas** [comment, marked as answer]
O aldığın hata bir değerlendirme testi.  Yani hatayı sen yapıyorsun. Çözmek sizin göreviniz. Çözemezsen dünyanın sonu değil :) 20 puan eksik alırsın.

**Reply by ElifAltunsu** [reply]
Tamamdır, hocam. Çok teşekkür ediyorum. Sorunun satır boşluklarındaki tab boşlukları olduğunu yeni anlayabildim. Kusura bakmayın. İyi akşamlar.
